main.nf

```diff
@@ -156,12 +193,18 @@ workflow {
         | MD5_VALIDATE \
         | FILE_EXT_VALIDATE \
         | SHOWINF_VALIDATE \
-        | XMLVALID_VALIDATE
+        | XMLVALID_VALIDATE \
+            | mix( SYNAPSE_CHECK.out, MD5_VALIDATE.out, FILE_EXT_VALIDATE.out, SHOWINF_VALIDATE.out, XMLVALID_VALIDATE.out ) \
+            | map { it[2] }
+            | collect \
+            | CSV_OUTPUT
+            | SYNAPSE_STORE
+
 }
 
 // Utility Functions
 
 def parseJson(file) {
     def parser = new groovy.json.JsonSlurper()
     return parser.parseText(file.text)
-}
+}
```

Comment:
Don't forget the empty line at the end of every file. I recommend setting the following preference in VS Code so it's done automatically for you ([source](https://stackoverflow.com/a/44704969)): 

```json
{
    "files.insertFinalNewline": true
}
```